movieapp/src/pages/HomePage/HomePage.tsx

```diff
@@ -2,12 +2,20 @@ import { Navigation } from '../../components/Navigation/Navigation';
 import { SortElement } from '../../components/SortElement/SortElement';
 import { MovieList } from '../../containers/MovieList/MovieList';
 
-export function HomePage(): JSX.Element {
+type MovieState = {
+    setMovies: Function;
+    movies: [];
```

Comment:
is it enough for TS to say that it's just array? 

let's talk about models

Reply:
Fixed